### ISSUE TITLE
TypeError: Cannot read property 'includes' of undefined

### ISSUE THREAD

**Issue by yedhrab** [tobiastimm/code-theme-converter#30]
```powershell
code2idea https://github.com/tobiastimm/raiju.git
Fetching repo
Converting 5 themes
🎉 Successfully converted the vscode theme for IDEA!
Converting Raijū (No italics)
Converting Raijū - Dark (No italics)
Converting Raijū - Classic
Converting Raijū
Converting Raijū - Dark
TypeError: Cannot read property 'includes' of undefined
```

**Comment by tobiastimm** [comment]
Will take a look into that

**Comment by tobiastimm** [comment]
@yedhrab should be fixed with v1.2.1
